File: Java/Booking/CheckTicketAvailabilityResult.java
package LTD_API;

public class CheckTicketAvailabilityResult
{
    private int ticketId;
    private boolean isAvailable;
    
    public int getTicketId()
    {
        return ticketId;
    }
    public void setTicketId(int ticketId)
    {
        this.ticketId = ticketId;
    }

    public boolean getIsAvailable()
    {
        return isAvailable;
    }
    public void setIsAvailable(boolean isAvailable)
    {
        this.isAvailable = isAvailable;
    }
}

File: PHP/Booking/CheckTicketAvailabilityResult.php
<?php
namespace LTD_API;

class CheckTicketAvailabilityResult
{
    private $ticketId;
    private $isAvailable;
    
    public function getTicketId()
    {
        return $this->ticketId;
    }
    public function setTicketId($ticketId)
    {
        $this->ticketId = $ticketId;
    }

    public function getIsAvailable()
    {
        return $this->isAvailable;
    }
    public function setIsAvailable($isAvailable)
    {
        $this->isAvailable = $isAvailable;
    }
}

